frog/imports/client/Wiki/helpers.js

```diff
@@ -105,12 +105,35 @@ const listWikis = async () => {
   });
 };
 
+// Function to get the pages of all pages in a wiki, returns [Title, ID]
+const listPages = async wikiDoc => {
+  const list = await new Promise(resolve =>
```

Comment:
No need to async/await for this function, we can simply return the Promise.

Reply:
Made the respective changes